templates/vue/src/components/SettingsModal.vue

```diff
@@ -213,12 +255,21 @@ export default {
       defaultDepth: 3,
       isExporting: false,
       renderImages: true,
+      renderMap: false,
+      mapBounds: { neLat: 90, neLng: 180, swLat: -90, swLng: -180 },
       hasExported: false,
     }
   },
   computed: {
     ...mapGetters(["tapestryJson"]),
     ...mapState(["settings", "rootId"]),
+    // the + before this.mapBounds changes type from string to int for correct comparison
+    latitudeRangeValid() {
+      return (+this.mapBounds.neLat || 90) > (+this.mapBounds.swLat || -90)
+    },
+    longitudeRangeValid() {
+      return (+this.mapBounds.neLng || 180) > (+this.mapBounds.swLng || -180)
+    },
```

Comment:
You probably also want to check that latitude and longitude are valid values / ranges. I entered `1000` and it accepted it!